tests/test-main.js

```diff
@@ -0,0 +1,24 @@
+var tests = [];
+for (var file in window.__karma__.files) {
+  if (window.__karma__.files.hasOwnProperty(file)) {
+    if (/test-.*?\.js$/.test(file) && !/test-main\.js/.test(file)) {
+      tests.push(file);
+    }
+  }
+}
+
+requirejs.config({
+  // Karma serves files from '/base'
+  baseUrl: '/base/lib/fxpay/',
+
+  paths: {
+    'helper': '../../tests/helper',
+    'es6-promise': '/base/lib/bower_components/es6-promise/promise',
```

Comment:
It looks like the ``es6-promise`` lib will call ``define()`` when it exists so can't we load this dynamically? It seems cumbersome to have to load all libs like this.